### PR TITLE
[Dashboard]Refine the presentation of Optimizing.

### PR DESCRIPTION
**Pull request by shidayang**
## Why are the changes needed?

Add startTime, Duration, Type, Task information in Optimizing page

## Brief change log

-

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by shidayang** [comment]
Before
image

After
image

File: ams/server/src/main/java/com/netease/arctic/server/dashboard/PaimonTableDescriptor.java [hunk 1]
```diff
@@ -169,23 +174,32 @@ public ServerTableMeta getTableDetail(AmoroTable<?> amoroTable) {
...
+  public List<AmoroSnapshotsOfTable> getSnapshots(
+      AmoroTable<?> amoroTable, String ref, OperationType operationType) {
     FileStoreTable table = getTable(amoroTable);
     List<AmoroSnapshotsOfTable> snapshotsOfTables = new ArrayList<>();
     Iterator<Snapshot> snapshots;
-    try {
-      snapshots = table.snapshotManager().snapshots();
-    } catch (IOException e) {
-      throw new RuntimeException(e);
+    if ("main".equals(ref)) {
```
Comment (HuangFru) [inline ams/server/src/main/java/com/netease/arctic/server/dashboard/PaimonTableDescriptor.java:182]: I think magic words could be avoided.

Comment (zhoujinsong) [reply ams/server/src/main/java/com/netease/arctic/server/dashboard/PaimonTableDescriptor.java:182]: Yes, maybe define a static value for that.

File: ams/server/src/main/java/com/netease/arctic/server/dashboard/model/OperationType.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.server.dashboard.model;
+
+import java.util.Arrays;
+
+public enum OperationType {
```
Comment (zhoujinsong) [inline ams/server/src/main/java/com/netease/arctic/server/dashboard/model/OperationType.java:23]: Add some Javadoc for the new class.

File: ams/server/src/main/java/com/netease/arctic/server/dashboard/PaimonTableDescriptor.java (hunk 1 above)
Comment (shidayang) [reply ams/server/src/main/java/com/netease/arctic/server/dashboard/PaimonTableDescriptor.java]: sure

**Review by zhoujinsong — APPROVED** [review]
LGTM.

**Pull request merged**